frontend/src/scenes/cohorts/Cohorts.tsx

```diff
@@ -142,7 +142,7 @@ export function Cohorts(): JSX.Element {
                 productKey={ProductKey.COHORTS}
                 thingName="cohort"
                 description="Use cohorts to group people together, such as users who used your app in the last week, or people who viewed the signup page but didn’t convert."
-                isEmpty={cohorts?.length == 0 && !cohortsLoading}
+                isEmpty={cohorts.count == 0 && !cohortsLoading && !searchTerm}
```

Comment:
I don't want to show the "create new cohort" option if the cohort list has filters applied